### PR TITLE
Fixredirect

### PR DESCRIPTION
**Pull request by schneider-clarisse**

### CODE REVIEW

File: src/js/organizer.client.jsx [hunk 1]
```diff
@@ -21,26 +21,17 @@ require('!style!css!sass!../sass/main.scss');
 const apiBaseURL = UrlService.getUrl();
 
 const app = document.getElementById('app');
-const redirectCallback = () => {
-  redirectAfterLogin();
-};
+const afterLogin = () => {
+  AppHistory.push('/')
```
Comment (StevenLocke) [inline src/js/organizer.client.jsx:25]: Does this need to be `/admin` for production?

File: src/js/organizer.client.jsx [hunk 2]
```diff
@@ -21,26 +21,17 @@ require('!style!css!sass!../sass/main.scss');
...
+}
 
 // onEnter callback to validate authentication in private routes
-const requireAuth = (nextState) => {
+const requireAuth = (nextState, replace) => {
```
Comment (StevenLocke) [inline src/js/organizer.client.jsx]: replace is never used, is it necessary here?

**Review by StevenLocke — APPROVED** [review]
There's a weird bug where localStorage doesn't always get cleared after clicking logout. It redirects to /login but doesn't clear your user or anything. I don't know if it's related, but I noticed it here since the sidebar changes way more dramatically.

File: src/js/organizer.client.jsx (hunk 1 above)
Comment (schneider-clarisse) [reply src/js/organizer.client.jsx:25]: maybe?

File: src/js/organizer.client.jsx (hunk 2 above)
Comment (schneider-clarisse) [reply src/js/organizer.client.jsx]: nah

Comment (larajanecka) [reply src/js/organizer.client.jsx]: linters are great for this.

File: src/js/organizer.client.jsx (hunk 1 above)
Comment (StevenLocke) [reply src/js/organizer.client.jsx:25]: Might help with #106 ?

**Comment by schneider-clarisse** [comment]
unable to repro the lack of localstorage clear, but we can't clear it in requireAuth because we set a logging_in cookie

**Comment by larajanecka** [comment]
Not sure if its a thing for this pr, but it looks like logging out of the attendee effects the side bar, but logging out of the organizer doesnt.

**Comment by schneider-clarisse** [comment]
.... ah yes, it should be. uno momento, another pr on the way. #derp

**Pull request merged**